src/tile.cpp

```diff
@@ -587,7 +587,7 @@ ReturnValue Tile::queryAdd(int32_t, const Thing& thing, uint32_t, uint32_t flags
 				return RETURNVALUE_NOTPOSSIBLE;
 			}
 
-			const Tile* playerTile = player->getTile();
+			auto playerTile = player->getTile();
```

Comment:
Could this be `const auto`
```suggestion
			const auto playerTile = player->getTile();
```